doc/source/jailtypes.rst

```diff
@@ -1,76 +1,102 @@
-==========
-Jail types
+.. index:: Jail Types
+.. _Jail Types:
+
+Jail Types
 ==========
 
 iocage supports five different jail types:
 
-* thick (default)
-* thin
-* base
-* template
-* empty
+* Full (default)
+* Clone (lightweight)
+* Basejail
+* Template
+* Empty
+
+All jail types have specific benefits and drawbacks, serving a variety
+of unique needs. This section describes and has creation examples for
+each of these jail types.
 
-All types have their pros & cons and serves different needs.
+.. index:: Full Jails
+.. _Full:
 
-Full (thick)
-------------
+Full
+----
 
-Full (thick) jail is the default type and it is created with the following command:
+The default **full** type jail is created with this command:
 
-``iocage create``
+:samp:`# iocage create -r 11.0-RELEASE`
 
-A full jail has a fully independent ZFS dataset suitable for network replication
-(ZFS send/recv).
+A **full** jail contains a fully independent ZFS dataset suitable for
+network replication (ZFS send/recv).
 
-Clone (thin)
-------------
+.. index:: Clone Jails
+.. _Clone:
 
-Thin jails are lightweight clones created with:
+Clone (lightweight)
+-------------------
 
-``iocage create -c``
+Clone jails are created with:
 
-Thin jails are cloned from the appropriate RELEASE at creation time and consume
-only a fraction of space, preserving only the changing data.
+:samp:`# iocage create -r 11.0-RELEASE -c 2`
 
-Base
-----
+Clone jails are duplicated from the appropriate RELEASE at creation
+time. These consume a small amount of space, preserving only the
+changing data.
 
-The original basejail concept based on nullfs mounts got popularized by ezjail.
-iocage basejails use independent read-only ZFS filesystem clones to achieve the
-same functionality.
+.. index:: Basejails
+.. _Basejail:
 
-To create a basejail execute:
+Basejail
+--------
 
-``iocage create -b``
+The original basejail concept was based on nullfs mounts. It was
+popularized by `ezjail <http://erdgeist.org/arts/software/ezjail/>`_,
+but :command:`iocage` basejails are a little different. Basejails in
+:command:`iocage` use independent read-only ZFS filesystem clones to
```

Comment:
We no longer use zfs clones for basejails. Instead they are mounts inside the jails fstab that get mounted on the jails startup.